src/compas_rhino/conduits/base.py

```diff
@@ -36,6 +37,17 @@ def enabled(self):
         finally:
             self.disable()
 
+    @property
+    def boundingbox(self):
+        if self.boundgbox is None:
+            boundingbox = self.CalculateBoundingBox()
+            self._boundingbox = boundingbox
+        return self._boundingbox
+
+    def CalculateBoundingBox(self, e):
+        bbox = Rhino.Geometry.BoundingBox(-1000, -1000, -1000, 1000, 1000, 1000)
+        e.IncludeBoundingBox(bbox)
+
```

Comment:
sorry, i misunderstood. since you have to define the bounding box by overwriting `CalculateBoundingBox`, there is not much point to the `self._boundingbox` attribute and corresponding property. you can remove that part again. the rest looks fine...